lib/ci/test_reporters/xunit_reporter.js

```diff
@@ -1,9 +1,10 @@
-var strutils = require('../../strutils')
+var displayutils = require('../../displayutils')
 var XmlDom = require('xmldom')
 
 
 function XUnitReporter(silent, out){
   this.out = out || process.stdout
+  this.intermediateOut = this.out == process.stdout ? process.stderr : process.stdout
```

Comment:
Could we disable the `intermediateOut` when stdout is already used? I don't like the idea printing to `stderr` when everything is fine.